### PR TITLE
Embroider blog post write up

### PR DESCRIPTION
**Pull request by MinThaMie**
## Ember Times review template

- [x] Add your name to top and bottom
- [x] Put emoji in writeup title
- [x] Add a short blurb (could be the title) to the beginning
- [x] Link to external article/repo/etc in paragraph/body text, not just the writeup title (link in writeup title is optional now)
- [x] Add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account
- [x] Check that all links work

### CODE REVIEW

File: content/the-ember-times-issue-180.md [hunk 1]
```diff
@@ -50,15 +52,13 @@ Check out Fleetbase on [ProductHunt](https://www.producthunt.com/posts/fleetbase
...
 ---
 
-## [3. Section title in sentence case 🐹](section-url)
+## [Embroider: from zero to route splitting in 3.5 weeks 📝](https://dev.to/bendemboski/embroider-from-zero-to-route-splitting-in-3-5-weeks-5abo)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+If you have been part of the Ember community for a while now and have read some of the latest Ember Times you have heard about [Embroider](https://github.com/embroider-build/embroider). This project is currently at version 0.41.0 and usable in Ember projects.  [Ben Demboski @bendemboski](https://github.com/bendemboski) wrote this [blog](https://dev.to/bendemboski/embroider-from-zero-to-route-splitting-in-3-5-weeks-5abo) about how they implemented Embroider in their [`ember-electron`](https://ember-electron.js.org) app.
```
Comment (ijlee2) [inline content/the-ember-times-issue-180.md]: I think it'd be good to link to last week's issue, as Amy had worked on a Reader's question. What do you think about introducing Ben in a new paragraph?

```suggestion
If you have been part of the Ember community for a while now and have read [The Ember Times' last issue](https://blog.emberjs.com/the-ember-times-issue-179#toc_readers-question-what%E2%80%99s-the-current-status-of-the-embroider-project-%F0%9F%A7%B5), you have probably heard about [Embroider](https://github.com/embroider-build/embroider). This project is currently at version 0.41.0 and usable in Ember projects.

[Ben Demboski @bendemboski](https://github.com/bendemboski) wrote this [blog](https://dev.to/bendemboski/embroider-from-zero-to-route-splitting-in-3-5-weeks-5abo) about how they implemented Embroider in their [`ember-electron`](https://ember-electron.js.org) app.
```

File: content/the-ember-times-issue-180.md [hunk 2]
```diff
@@ -50,15 +52,13 @@ Check out Fleetbase on [ProductHunt](https://www.producthunt.com/posts/fleetbase
...
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+This [blogpost](https://dev.to/bendemboski/embroider-from-zero-to-route-splitting-in-3-5-weeks-5abo) explains in detail the app and the process they used to make their app use Embroider. All the hard things they ran into like ES6 module compliance and third-party add-ons. Luckily after 3.5 weeks they achieved what they wanted: **tree-shaking** and **code splitting across routes**. So if you want to start taking advantage of what Embroider has to offer, this post is a great place to start.
```
Comment (ijlee2) [inline content/the-ember-times-issue-180.md]:
```suggestion
Ben's [blog post](https://dev.to/bendemboski/embroider-from-zero-to-route-splitting-in-3-5-weeks-5abo) explains in detail the app and the process they used to make their app use Embroider. All the hard things they ran into, such as ES6 module compliance and third-party add-ons. Luckily, after 3.5 weeks, they achieved what they wanted: **tree-shaking** and **code splitting across routes**. So if you want to start taking advantage of what Embroider has to offer, this post is a great place to start.
```

**Review by ijlee2 — APPROVED** [review]
Awesome, thanks for covering Ben's Embroider blog post!

File: content/the-ember-times-issue-180.md (hunk 1 above)
Comment (MinThaMie) [reply content/the-ember-times-issue-180.md]: Great suggestion :)

**Comment by bendemboski** [comment]
👍 

FYI I use he/him pronouns, but don't mind they/them either, so feel free to leave or change as you prefer.

**Comment by ijlee2** [comment]
@bendemboski Ah, thank you for letting us know. I can update the pronoun in the `blog/embertimes-180`.

**Pull request merged**